client/app/components/routes/select-operator/form/UserOperator.tsx

```diff
@@ -107,19 +106,9 @@ export default async function UserOperator({
     // If the operator exists then show the form from the second page
     <UserOperatorContactForm formData={formData} schema={userOperatorPage2} />
   ) : (
-    <>
-      {/* This note should only show up when a user is editing an existing operation. However, that page isn't built yet, so for now I'm showing it here so that designers can check. A conditional will be added once the page exists. */}
-      <Note
-        classNames={"mb-4 mt-6"}
-        showNotePrefix={false}
-        showAlertIcon={true}
-        message="Please fill out missing information or update incorrect information about your operator to continue with your application.
-Some fields cannot be edited. If you need to change those fields, please contact us via email at GHGRegulator@gov.bc.ca."
-      />
-      <UserOperatorMultiStepForm
-        schema={createUserOperatorSchema(businessStructuresList)}
-        formData={formData}
-      />
-    </>
```

Comment:
I did know about this as Brianna and Nan have been discussing it and was going to suggest doing it in a separate PR but decided to squash this now.